src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+  const discountOne = 50;
+  const discountTwo = 20;
+  const cost = days * rentCost;
+
+  if (days < 3) {
```

Comment:
Create a variable for this magic number

Reply:
Still not fixed from the previous review
<img width="849" alt="image" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/64750958/5d9a769e-9958-4cf3-952a-9096a1bc3290">
```suggestion
  const shortTerm = 3;

  if (days < shortTerm) {
```